cdap-common/bin/functions.sh

```diff
@@ -1088,11 +1084,12 @@ cdap_upgrade_tool() {
 }
 
 #
-# cdap_setup_tool [arguments]
+# cdap_hbase_setup [arguments]
 #
-cdap_setup_tool() {
+cdap_hbase_setup() {
   local readonly __ret __class=co.cask.cdap.data.tools.CoprocessorBuildTool
 
+  logecho "$(date) Ensuring required HBase coprocessors are on HDFS"
```

Comment:
Move this up to where we're calling `cdap_hbase_setup` in `cdap_start_java` so we don't write to the log when called on the command line.